politeiawww/cmd/pictl/print_test.go

```diff
@@ -0,0 +1,29 @@
+package main
```

Comment:
When you add a new file to the repo, start the file with the copyright statement.

```
// Copyright (c) 2021 The Decred developers
// Use of this source code is governed by an ISC
// license that can be found in the LICENSE file.

package main
```